configs/internal_flash_no_rot.json

```diff
@@ -160,6 +160,21 @@
             "update-client.storage-address"             : "(MBED_ROM_START + MBED_BOOTLOADER_FLASH_BANK_SIZE)",
             "kvstore-size"                              : "(2*32*1024)",
             "storage_tdb_internal.internal_base_address": "(MBED_ROM_START+MBED_BOOTLOADER_SIZE)"
+        },
+        "MIMXRT1050_EVK": {
+            "target.macros_remove"                      : ["HYPERFLASH_BOOT"],
+            "target.mbed_rom_start"                     : "0x60000000",
```

Comment:
Not possible to get these from a CMSIS-Pack or from the targets.json-file? If not then this is fine by me.

Reply:
I think I checked this before and these are not extracted succesfully from a CMSIS-pack, I am not sure if the extrenal ROM has been defined there at all. These could be defined in the `target.json` though.

Reply:
Does the address depending on if you're using HyperFlash or QSPI-flash?`
If it does, we might have to have 2 different targets.